### DISCUSSION TITLE
Custom error code resolver issue.

### DISCUSSION THREAD

**Discussion by jiancqiu** [TongchengOpenSource/smart-doc#461] [Q&A]
I implemented a custom error resolver based on the documentation, but it's not effective. I hope to get some help. Here's the code:
通过文档自己实现了一下自定义错误解析器，但无法生效，希望能够得到帮助，代码如下：

`public class EExceptionValuesResolver implements DictionaryValuesResolver {
    
    @NotNull
    @Override
    public <T extends EnumDictionary> Collection<T> resolve() {
        List<EnumDictionary> list = new ArrayList<>();
        EnumDictionary enumDictionary = new EnumDictionary();
        // Reflect on handling your own error code, and fill in the return.
        Map<String, String> errorMap = FileResolve.propertiesReader();

        Set<String> keys = errorMap.keySet();
        for(String key:keys){
            enumDictionary.setValue(key);
            enumDictionary.setDesc(errorMap.get(key));
        }
        list.add(enumDictionary);
        return (Collection<T>) list;
    }
}`

----------------------------------config-----------------------------------------
"errorCodeDictionaries": [

  {
    "title": "error code",
    "valuesResolverClass": "com.capgemini.mail.constant.EExceptionValuesResolver"
  }
]

**Comment by chenqi146** [comment]
Are there any related error messages?
有相关的报错信息吗
